src/type/Type.h

```diff
@@ -456,6 +456,7 @@ DEFINE_TYPE_DETECT(char*, VARCHAR, StringType, std::string_view, "")
 DEFINE_TYPE_DETECT(std::string_view, VARCHAR, StringType, std::string_view, "")
 DEFINE_TYPE_DETECT(std::string, VARCHAR, StringType, std::string_view, "")
 DEFINE_TYPE_DETECT(void*, INVALID, StructType, void*, nullptr)
+DEFINE_TYPE_DETECT(uint64_t, INVALID, Int128Type, uint64_t, 0)
```

Comment:
Add this because the arg in TPM is 'long unsigned int'

Reply:
We probably don't want this - as the C++ type for each Nebula Type should be fixed.
But could you explain the problem `because the arg in TPM is 'long unsigned int'`?

Reply:
Removed.
sorry for the confusion, I was trying to get the input type of input parameters in UDAF, TPM passed in threshold parameter with size_t, which is used as 'long unsigned int' when pass to `DEFINE_TYPE_DETECT`

Reply:
I see - that is not input type, :) input type implies the column type of the data to aggregate.